src/homepageExperience/components/steps/go/ExecuteQuerySql.tsx

```diff
@@ -33,7 +27,10 @@ query := \`SELECT *
           WHERE time >= now() - interval '1 hour'
             AND ('bees' IS NOT NULL OR 'ants' IS NOT NULL)\`
 
-iterator, err := client.Query(context.Background(), "${bucket}", query)
+queryOptions := influxdb3.QueryOptions{
+  Database: database,
```

Comment:
I see that `bucket` is being removed but then how is this `database` variable populated?

Reply:
The `database` is initialised in the "write-data" step: https://github.com/influxdata/ui/blob/a60db0a94c077ff49751490115a109cbc1df0400/src/homepageExperience/components/steps/go/WriteDataSql.tsx#L95